VideoLocker/build.gradle

```diff
@@ -275,7 +275,6 @@ android {
     testOptions { 
          unitTests.all {
              maxHeapSize '1g'
-             jvmArgs '-XX:MaxPermSize=512m'
```

Comment:
OK. You seem to have missed importing these changes to the new pull request though.